pwa/src/styles/main.css

```diff
@@ -227,3 +227,7 @@ a:hover {
 .utrecht-button:disabled:hover {
     cursor: not-allowed;
 }
+
+.select-elementCreation-button {
+    margin-top: 28px;
+}
```

Comment:
Simply by removing the `label` and removing these `css` rules. The layout would look like this:

![image](https://user-images.githubusercontent.com/66552932/153555123-1420b911-f219-41c8-becb-f150c206bf09.png)

Also, this way of positioning something can be dangerous. You're positioning the element from within itself, based on outside context (e.g. the button has to go down, because the outside context - in this case the select field - is lower than itself so it doesn't align). What would happen when we want to include this same button somewhere else, but in another context? It would get an unintentional `margin-top`, most likely breaking layout. Besides wanting to re-use this button, what would happen when the context changes? 

E.g.: the padding of the label changes, than the following happens:

![image](https://user-images.githubusercontent.com/66552932/153555479-c1fcbdac-09b8-4804-90ad-6b5435e6678e.png)

In this case, we can just remove the label, so it doesn't really matter. In other cases however, we could have wrapped the `select` and `input` components in a `div` and set the position of those elements **within** that `div`, removing outside context from the elements itself.